test/performance/browsing_test.rb

```diff
@@ -1,12 +1,12 @@
-#require 'test_helper'
-#require 'rails/performance_test_help'
+require 'test_helper'
+require 'rails/performance_test_help'
 
-#class BrowsingTest < ActionDispatch::PerformanceTest
-  # Refer to the documentation for all available options
-  # self.profile_options = { :runs => 5, :metrics => [:wall_time, :memory]
-  #                          :output => 'tmp/performance', :formats => [:flat] }
+class BrowsingTest < ActionDispatch::PerformanceTest
+   #Refer to the documentation for all available options
+   #self.profile_options = { :runs => 5, :metrics => [:wall_time, :memory]
```

Comment:
Should these still be commented out?

Reply:
I believe tests run with a default setups. If we would like to configure the test runs, we can uncomment the section and specify our own values.